### PR TITLE
Сreate Header component

### PR DESCRIPTION
**Pull request by alex6061**

### CODE REVIEW

File: js/components/widgets/Header/Header.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/**@typedef {import('./types').Header} Header */
+
+/**
+ * @function Header
+ * @param {Object} data 
+ * @returns {string}
+ */
+
+export const Header = (data) => {
+  if (Object.keys(data) !== 5) return '';
```
Comment (nagorapro) [inline js/components/widgets/Header/Header.js:10]: Bad

File: js/components/widgets/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/**@typedef {import('./types').Header} Header */
+
+/**
+ * @function Header
+ * @param {Object} data
```
Comment (nagorapro) [inline js/components/widgets/Header/Header.js:5]: Bad

File: js/components/widgets/Header/Header.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/**@typedef {import('./types').Header} Header */
```
Comment (nagorapro) [inline js/components/widgets/Header/Header.js:1]: SPACE

File: js/components/widgets/Header/types.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export type Header = {
+  name: string;
+  logo: string;
+  menuItems: string;
```
Comment (nagorapro) [inline js/components/widgets/Header/types.ts:4]: Bad

File: js/components/widgets/Header/types.ts [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+  lang: [],
```
Comment (nagorapro) [inline js/components/widgets/Header/types.ts:5]: Bad

File: js/components/widgets/Header/types.ts [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+  theme: [],
```
Comment (nagorapro) [inline js/components/widgets/Header/types.ts:6]: Bad

File: js/data.js [hunk 7]
```diff
@@ -1,11 +1,14 @@
...
     header: {
+      name: 'header',
       logo: {
         source: './../assets/img/1.jpg',
         description: 'logo',
       },
-    },
+      menuItems:'',
+      lang: [],
+      theme: '',
```
Comment (nagorapro) [inline js/data.js:11]: Bad

File: js/data.js [hunk 8]
```diff
@@ -1,11 +1,14 @@
 export const data = {
   en: { 
     header: {
+      name: 'header',
       logo: {
         source: './../assets/img/1.jpg',
```
Comment (nagorapro) [inline js/data.js:6]: Bad

File: js/data.js [hunk 9]
```diff
@@ -30,6 +33,7 @@ export const data = {
         description: 'download__image',
```
Comment (nagorapro) [inline js/data.js:33]: Bad

**Review by nagorapro — COMMENTED** [review]
Need fix ( sorry:) )

File: js/components/widgets/Header/Header.js [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+/**
+ * @function Header
+ * @param {Data} data 
+ * @returns {string}
+ */
+
+export const Header = (data) => {
+
+  const dataValue = Object.values(data);
```
Comment (nagorapro) [inline js/components/widgets/Header/Header.js:11]: dataValueSSSSS

File: js/components/widgets/Header/Header.js [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+
+  if (dataValue.length !== 5 ) return '';
+
+  const {name, logo, menuItems, theme, lang} = data;
+
+  const className = name ? name : 'header';
+
+  return `
+    <header className="${className}">
```
Comment (nagorapro) [inline js/components/widgets/Header/Header.js:20]: Use class attribute
Its html

File: js/components/widgets/Header/Header.js [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+      logo...
+      navigation...
+      theme...
+      lang...
```
Comment (nagorapro) [inline js/components/widgets/Header/Header.js:24]: ???

**Review by nagorapro — COMMENTED** [review]
Need fix

**Comment by alex6061** [comment]
delete branch

**Comment by alex6061** [comment]
deleted branch

**Pull request closed without merging**